source/aws_profile.c

```diff
@@ -1214,7 +1214,7 @@ static void s_parse_and_apply_line_to_profile_collection(
         return;
     }
 
-    AWS_LOGF_WARN(AWS_LS_SDKUTILS_PROFILE, "Unidentifiable line type encountered while parsing profile file");
+    AWS_LOGF_DEBUG(AWS_LS_SDKUTILS_PROFILE, "Unidentifiable line type encountered while parsing profile file");
```

Comment:
Again fatal error and should not occur frequently. 
```suggestion
    AWS_LOGF_ERROR(AWS_LS_SDKUTILS_PROFILE, "Unidentifiable line type encountered while parsing profile file");
```